platform/windows/dependencies.ps1

```diff
@@ -0,0 +1,16 @@
+Write-Host "Install csound dependencies"
+
+choco install -y winflexbison3 innosetup llvm javaruntime jdk8 swig python
```

Comment:
What is llvm needed for?

Reply:
It's not needed.  I removed it from the install script.